docker-compose.yml

```diff
@@ -79,7 +79,7 @@ services:
       - mysql:/var/lib/mysql
 
     # For compatibility with PHP versions that don't support the caching_sha2_password auth plugin used in MySQL 8.0.
-    command: --default-authentication-plugin=mysql_native_password
+    command: ${LOCAL_DB_AUTH_OPTION-}
```

Comment:
Is the `-` at the end intentional?

Reply:
When these are substituted, anything after the `-` is considered the default if `LOCAL_DB_AUTH_OPTION` is empty. I'm unsure if excluding it will cause an error if `LOCAL_DB_AUTH_OPTION` is missing. Let me do some testing.

Reply:
> WARN[0000] The "LOCAL_DB_AUTH_OPTION" variable is not set. Defaulting to a blank string.
Looks like this warning happens without the trailing `-` when the variable is not defined. Confirmed that leaving it in serves as a blank default.